src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/onetoplevelclass/InputOneTopLevelClassAnnotation.java

```diff
@@ -0,0 +1,5 @@
+package com.puppycrawl.tools.checkstyle.checks.design.onetoplevelclass;
+
+@interface InputOneTopLevelClassAnnotation {
```

Comment:
please add `// violation` comments in test inputs on lines where violation is reported

Reply:
Fixed